imageio/plugins/ffmpeg.py

```diff
@@ -771,10 +771,10 @@ def _initialize(self):
                 if self._size[0] % macro_block_size > 0:
                     out_h += macro_block_size - \
                         (self._size[0] % macro_block_size)
-                cmd += ['-vf', 'scale={0}:{1}'.format(out_w, out_h)]
+                cmd += ['-vf', 'scale={}:{}'.format(out_w, out_h)]
                 logging.warning(
                     "IMAGEIO FFMPEG_WRITER WARNING: input image is not"
-                    " divisible by macro_block_size={0}, resizing from {1} "
+                    " divisible by macro_block_size={}, resizing from {} "
                     "to {2} to ensure video compatibility with most codecs "
```

Comment:
You missed a spot; I think this is what makes Travis fail right now.

Reply:
Thanks, fixed!

I normally enable Travis CI and check it passes before submitting PRs, but in this case I see it's been explicitly disabled in .travis.yml:

```yml
# Only run travis build on push for master branch
branches:
  only:
    - master
```
In general, I'd advise against this for exactly this case, it prevents feature branches form being automatically tested too.